examples/global-external-auth/01-authserver.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testserver
```

Comment:
this'll need a `namespace: projectcontour` as well since that's where the ExtensionService is..